lld/test/wasm/unsupported-pic-relocations.s

```diff
@@ -0,0 +1,39 @@
+# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown -o %t.o %s
+
+# RUN: not wasm-ld --experimental-pic -shared %t.o -o /dev/null 2>&1 | \
+# RUN:   FileCheck %s
+
+# RUN: not wasm-ld --experimental-pic -shared %t.o -o /dev/null  --unresolved-symbols=report-all 2>&1 | \
+# RUN:   FileCheck %s
+
+# RUN: not wasm-ld --experimental-pic -shared %t.o -o /dev/null  --warn-unresolved-symbols 2>&1 | \
+# RUN:   FileCheck %s
+
+# RUN: not wasm-ld --experimental-pic -shared %t.o -o /dev/null  --unresolved-symbols=ignore-all 2>&1 | \
+# RUN:   FileCheck %s
+
+# RUN: not wasm-ld --experimental-pic -shared %t.o -o /dev/null  --unresolved-symbols=import-dynamic 2>&1 | \
+# RUN:   FileCheck %s
+
+.functype external_func () -> ()
+
+use_undefined_function:
+    .functype use_undefined_function () -> ()
+    i32.const external_func@TBREL
```

Comment:
I'm still curious how you were able to generate such relocations?  IIUC, LLVM will not generate these relocations for undefined symbols.  Did you come across this issue by writing asm code?  Or is there some issue where LLVM can generate these wrongly?

Reply:
I was using Rust and emitted LLVM IR that I manually compiled down to a Wasm object file.  During testing, I once missed a LLVM IR input file and then saw those relocations on unused functions, without getting an error. It is certainly an edge case.